napari/layers/image/_image_constants.py

```diff
@@ -93,7 +93,7 @@ class ImageRendering(StringEnum):
     * additive: voxel colors are added along the view ray until
       the result is saturated.
     * iso: isosurface. Cast a ray until a certain threshold is
-      encountered. At that location, lighning calculations are
+      encountered. At that location, lightning calculations are
```

Comment:
```suggestion
      encountered. At that location, lighting calculations are
```
Typo, but wrong fix.